src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2639,3 +2639,8 @@ Redownload\ missing\ files=Redownload missing files
 Redownload\ missing\ files\ for\ current\ library?=Redownload missing files for current library?
 
 File\ directory\ needs\ to\ be\ set\ or\ does\ not\ exist.\nPlease\ save\ your\ library\ and\ check\ the\ preferences.=File directory needs to be set or does not exist.\nPlease save your library and check the preferences.
+
+Note\:\ The\ study\ directory\ should\ be\ empty.=Note: The study directory should be empty.
```

Comment:
I think, the UI pattern of JabRef is to use colors to indicate "Note" and "Warning".

OK, we do have some strings with "Warning:" in front. Too much work to discuss this now. I'll just merge and we can do a follow-up for consistent UI later.

Reply:
Sure.